src/apis/types/recurring_types.ts

```diff
@@ -349,7 +349,7 @@ export type ChargeEvent = {
 };
 
 /** Refund charge request */
-export type ModifyCharge = {
+export type ChargeModification = {
```

Comment:
Why not "ChargeBody"? Or is that leaking the abstraction too heavily in the name?

Reply:
Hmm. Looking at EPayment, the equivalent is "EPaymentModificationRequest". I'll update the types here to be consistent with EPayment